v2/periphery/LockupLinearBatchStreamCreator.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.8.13;
```

Comment:
Needs to be `>=0.8.19`.

Reply:
Both work, I just followed you

https://github.com/sablier-labs/examples/blob/76957062a57b1cb90b4891ae7c2abbfb8034dc99/v2/core/LockupLinearStreamCreator.sol#L2

https://github.com/sablier-labs/examples/blob/76957062a57b1cb90b4891ae7c2abbfb8034dc99/v2/core/LockupDynamicStreamCreator.sol#L2

Reply:
I know - I made a mistake.

https://github.com/sablier-labs/examples/issues/6

Reply:
Fixed via https://github.com/sablier-labs/examples/pull/4/commits/a508d1063dad56632f2a7f66e74715bb0ca13369.